src/utils/__tests__/index.ts

```diff
@@ -83,6 +83,8 @@ jest.mock(
 );
 jest.mock('~/context', require('~/testUtils/mocks/polkadot').mockContextModule('~/context'));
 
+const encodedAddressStub = sinon.stub(encodeAddressModule, 'default');
```

Comment:
Better go back to stubbing in each of the tests. To bypass the sinon error, in both test suites (inside the describe but outside the tests), add an afterAll that calls sinon.restore()